lib/Model/Member.php

```diff
@@ -63,6 +63,7 @@ public function inviteToCircle($circleType) {
 	public function joinCircle($circleType) {
 
 		switch ($circleType) {
+			case Circle::CIRCLES_PERSONAL:
```

Comment:
why one should be able to join a Personal Circle ?

Reply:
This is necessary when you want to add someone to your own personal circle when the `skip_invitation_to_closed_circles` option is enabled

Reply:
please remove this part, #416 fix this issue